fmf.spec

```diff
@@ -70,10 +73,12 @@ install -pm 644 fmf.1* %{buildroot}%{_mandir}/man1
 %doc README.rst examples
 %license LICENSE
 
-%files -n python%{python3_pkgversion}-%{name}
-%{python3_sitelib}/%{name}/
-%{python3_sitelib}/%{name}-*.egg-info
+%files -n python3-fmf -f %{pyproject_files}
+# Epel9 does not tag the license file in pyproject_files as a license. Manually install it in this case
+%if 0%{?el9}
 %license LICENSE
```

Comment:
License is added by pyproject macros, just readme is needed.

Reply:
In `packit` we noticed that that was not the case for epel9. Can you test and confirm otherwise?

Reply:
I'd leave this until the conversation about manually creating python3-fmf is resolved.